### ISSUE TITLE
Using Categorical_crossentropy, Loss Unable to Converge

### ISSUE THREAD

**Issue by MissLikeWind** [ellisdg/3DUnetCNN#94]
Hello, when using the softmax activation of output_layer and categorical_crossentopy loss,   it is unable to  converge. Do you have any advices? Thank you

**Comment by ellisdg** [comment]
For some reason, I have been unable to get the softmax activation to work
despite several attempts.

**Comment by Schnabbel** [comment]
In order to use softmax activation, you have to wrap it in a Lambda Layer and specify the keyword argument axis = 1

Like so:
`from keras.activations import softmax
    if activation_name == 'softmax':
        activation_block = Lambda(lambda x: softmax(x, axis=1))(output_layer)`
